x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/ltr/LearningToRankRescorerContext.java

```diff
@@ -72,6 +71,14 @@ List<FeatureExtractor> buildFeatureExtractors(IndexSearcher searcher) throws IOE
             featureExtractors.add(new QueryFeatureExtractor(queryFeatureNames, weights));
         }
 
+        List<String> fieldValueExtractorFields = this.regressionModelDefinition.inputFields()
+            .stream()
+            .filter(not(queryFeatureNames::contains))
+            .toList();
+        if (fieldValueExtractorFields.isEmpty() == false) {
+            featureExtractors.add(new FieldValueFeatureExtractor(fieldValueExtractorFields, this.executionContext));
+        }
+
```

Comment:
// Should probably be commented until we have the new field value as a standard extractor.